examples/gnrc_border_router/Makefile.board.dep

```diff
@@ -2,8 +2,11 @@
 ifeq (,$(filter native,$(BOARD)))
   ifeq (1,$(USE_SLIP))
     USEMODULE += slipdev
-  else
+  else ifeq (1,$(USE_ETHOS))
     USEMODULE += stdio_ethos
+  else ifeq (1,$(USE_ESP))
```

Comment:
```suggestion
  else ifeq (1,$(USE_WIFI))
    ifneq (,$(filter arch_esp%,$(FEATURES_PROVIDED))
```

was what I meant in https://github.com/RIOT-OS/RIOT/pull/13545/files#r387348463.

Alternatively (and probably more correct)

```suggestion
  else ifeq (1,$(USE_WIFI))
    ifneq (,$(filter esp32 esp8266,$(CPU))
```